esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -122,6 +130,9 @@
     ``{project}`` that vary between the different datasets will be transformed
     to something like  ``ambiguous_project``. Examples: ``title: 'Awesome Plot
     of {long_name}'``, ``xlabel: '{short_name}'``, ``xlim: [0, 5]``.
+time_format: str, optional (default: None)
+    Matplotlib strftime format string for the time axis, if None defaults to
+    time units of iris cube
```

Comment:
```suggestion
time_format: str, optional (default: None)
    :func:`~datetime.strftime` format string that is used to format the time
    axis using :class:`matplotlib.dates.DateFormatter`. If ``None``, use the
    default formatting imposed by the iris plotting function.
```

This adds some nice links to the documentation for more information